client/src/Main.elm

```diff
@@ -25,30 +25,60 @@ main =
 
 
 type Msg
-    = Outgoing Value
-    | Incoming String
+    = SetEditorContent String
+    | SetViewContent String
     | ChangeLanguage String
+    | SendEditorContent
 
 
 type alias Model =
-    String
+    { editorContent : String
+    , viewContent : String
+    , language : Language
```

Comment:
does this beong here ? Do model need to know about language?

Reply:
The Model is where you store your state. Otherwise you won't be able to remember the selected option.